### DISCUSSION TITLE
Make MIDI note names visible when editing MIDI clips

### DISCUSSION THREAD

**Discussion by on3iro** [helgoboss/helgobox#1111] [Ideas]
Hey,

I encountered another issue and am not sure, if this is a bug or if I am doing something wrong:
I just put the Sitala drum sampler into one of playtimes tracks. After recording a clip and opening the midi editor I don't see the sample names for the notes. However if I open the midi editor directly in Reaper note names are immediately shown correctly.

This an issue for me right now, as it is pretty hard to figure out where my sounds are currently placed, e.g. to edit in some Hi Hats.

**Comment by helgoboss** [comment]
This is one side effect of a change that was requested by another user during the alpha stage. Initially, I was putting the editable MIDI item on the column track (the one which would contain Sitala). In that case, you would see the note names. But in order to not clutter the arrangement view, I started putting the editable MIDI item on a temporary hidden track that contains a send to the instrument track. That's why the note names are not there.

Hmm, maybe I should make this configurable. The user can then decide between a completely uncluttered arrangement view and note names.

**Comment by helgoboss** [comment]
I will fix this as part of #1313 (by creating the MIDI items on the original column track).
